backtrace-library/src/main/java/backtraceio/library/BacktraceDatabase.java

```diff
@@ -47,6 +49,7 @@
     private DatabaseFileContext backtraceDatabaseFileContext;
     private BacktraceDatabaseSettings databaseSettings;
     private boolean _enable = false;
+    private BacktraceBreadcrumbs backtraceBreadcrumbs;
```

Comment:
User right now can't use his own implementation of breadcrumbs, so Breadcurmbs interface isn't that useful at all. I think it will be better if we will change object definition from ` private BacktraceBreadcrumbs backtraceBreadcrumbs;`  to `private Breadcrumbs backtraceBreadcrumsb`

Reply:
Done, thank you!